packages/token-balance-cache/src/provider.ts

```diff
@@ -118,21 +121,46 @@ export default class TokenBalanceProvider {
       process.env.TERRA_SETTEN_PHOENIX_API_KEY
     }`;
 
+    let bankBalance = new BN(0);
     try {
       // NOTE: terra.js staking module is incompatible with stargate queries
       const balResp = await axios.get(queryUrl); // [ { denom: 'uluna', amount: '5000000' } ]
       const balances: Array<{ denom: string, amount: string }> = balResp?.data?.balances;
-      let balance = new BN(0);
       if (balances?.length > 0) {
         const balanceObj = balances.find(({ denom }) => denom === 'uluna');
         if (balanceObj) {
-          balance = new BN(balanceObj.amount);
+          bankBalance = new BN(balanceObj.amount);
         }
       }
-      return balance;
     } catch (e) {
-      throw new Error(`no balance found: ${e.message}`);
+      log.warn(`could not fetch terra bank balance: ${e.message}`);
     }
+
+    // make staking query
+    let stakedBalance = new BN(0);
+    const stakedQueryUrl = `${url}/cosmos/staking/v1beta1/delegations/${
+      userAddress
+    }?key=${
+      process.env.TERRA_SETTEN_PHOENIX_API_KEY
```

Comment:
let's remove this and use the lcd phoenix url, as terra says it should be fine for prod 🤷